ivy_tests/test_ivy/test_functional/test_nn/test_losses.py

```diff
@@ -15,23 +15,17 @@
         max_value=1,
         allow_inf=False,
         min_num_dims=1,
-        max_num_dims=1,
-        min_dim_size=2,
     ),
     dtype_and_pred=helpers.dtype_and_values(
         available_dtypes=helpers.get_dtypes("float"),
         min_value=1e-04,
         max_value=1,
         allow_inf=False,
-        exclude_min=True,
-        exclude_max=True,
         min_num_dims=1,
-        max_num_dims=1,
-        min_dim_size=2,
     ),
     reduction=st.sampled_from(["none", "sum", "mean"]),
     axis=helpers.ints(min_value=-1, max_value=0),
```

Comment:
The axis value shouldn't be hardcoded, instead we should update the strategies to actually generate valid axis values.